docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -237,20 +239,26 @@ La configurazione del frontmatter `sidebar` viene utilizzata solo per i collegam
 
 I collegamenti possono anche includere una proprietà `badge` per visualizzare un badge accanto all'etichetta del collegamento.
```

Comment:
It looks like this is missing the reference to `groups` and `autogenerated groups` from the original translation.